bump.js

```diff
@@ -0,0 +1,21 @@
+const fs = require("fs");
+const path = require("path");
+const { exec } = require("@actions/exec");
+
+(async () => {
+  await exec("pnpm changeset", ["status", "--output=./.changeset/status.json"]);
+  await exec("pnpm changeset", ["version"]);
+  const changesetStatus = require("./.changeset/status.json");
+  changesetStatus?.releases.map((release) => {
+    const { name: appName } = release;
+    const newVersion = `${require(`./apps/${appName}/package.json`).version}`;
```

Comment:
Don't we need a way of checking that we're dealing with an app and not a a package here? I'm assuming changeset will bump both packages and apps if it deems it necessary.

Reply:
This should happen locally when you are generating the changeset 
![Screenshot 2022-08-24 at 14 45 16](https://user-images.githubusercontent.com/13068580/186410388-dfc51ef8-14ae-4ca7-ba77-fad9b8a7abd3.png)

Reply:
I see.
Any ideas/pattern that we currently use  for packages?